security-questionnaire.md

```diff
@@ -0,0 +1,73 @@
+In terms of technical capabilities, the proposed API is a DOM-based API. In
```

Comment:
I'd add a single (admittedly redundant) sentence on what it does. Maybe:

"In terms of technical capabilities, the proposed API is a DOM-based API: It turns a string or DOM fragment into another string or fragment."